TODO

```diff
@@ -19,8 +19,6 @@ API additions
 
 - Add hb-cairo glue
 
-- Add sanitize API (and a cached version, that saves result on blob user-data)
```

Comment:
Is the rest something I can do here or should I turn that into a GitHub issue?

Reply:
No, don't want the caching.